include/CLI/LeapFormatter.hpp

```diff
@@ -20,9 +20,9 @@ namespace CLI {
 
 class LeapFormatter : public Formatter {
    // pseudographics - to draw subcommand tree
-   const char* tree_line = u8"\u2502";
-   const char* tree_angle = u8"\u2514";
-   const char* tree_fork = u8"\u251C";
+   const char* tree_line = reinterpret_cast<const char *>(u8"\u2502");
+   const char* tree_angle = reinterpret_cast<const char *>(u8"\u2514");
+   const char* tree_fork = reinterpret_cast<const char *>(u8"\u251C");
```

Comment:
That was an interesting rabbit hole.
https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1423r2.html

Reply:
Yes, it is a little bit crazy that we have to resort to this!